app/controllers/admin/school_groups/users_controller.rb

```diff
@@ -73,7 +78,20 @@ def produce_csv(school_group, group_admins, school_users)
       def add_user_to_csv(csv, school_group, school, user)
         csv << [
           school_group.name,
-          school.present? ? school.name : 'N/A',
+          school&.name || '',
+          school&.school_type&.humanize || '',
+          if school
+            school&.active? ? 'Yes' : 'No'
+          else
+            ''
+          end,
+          if school
+            school&.data_enabled? ? 'Yes' : 'No'
+          else
+            ''
+          end,
```

Comment:
perhaps `school ? (school.data_enabled? ? 'Yes' : 'No') : ''`

Reply:
This is what I had originally, but Rubocop wants if statements:

```
[Correctable] Style/NestedTernaryOperator: Ternary operators must not be nested. Prefer `if` or `else` constructs instead.
```